### PR TITLE
ms1555: add work_alias_pi

### PR DESCRIPTION
**Pull request by MingtaoSun**
@scoen @danielbaker
add work_alias_pi

### CODE REVIEW

File: banjo/conf/schema.xml [hunk 1]
```diff
@@ -216,6 +216,7 @@
    <!-- link copy to work -->
    <field name="work_id" type="long" indexed="true" stored="true"/>
    <field name="work_pi" type="string" indexed="true" stored="true"/>
+   <field name="work_alias_pi" type="text_general" indexed="true" stored="true" multiValued="true"/>
```
Comment (danielbaker) [inline banjo/conf/schema.xml]: whats the diff between string and text_general?

**Comment by bsingh-nla** [comment]
:+1:

File: banjo/conf/schema.xml (hunk 1 above)
Comment (bsingh-nla) [inline banjo/conf/schema.xml]: why not use "alias_pi"?

Comment (MingtaoSun) [inline banjo/conf/schema.xml]: @bsingh-nla "alias_pi" is being used as a copy field. "work_alias_pi" is the copy's work's alias pi.

Comment (MingtaoSun) [inline banjo/conf/schema.xml]: @danielbaker 
http://stackoverflow.com/questions/7175619/apache-solr-string-or-text
String stores a word/sentence as an exact string without performing tokenization etc. Commonly useful for storing exact matches, e.g, for facetting.
Text typically performs tokenization, and secondary processing (such as lower-casing etc.). Useful for all scenarios when we want to match part of a sentence.

**Comment by danielbaker** [comment]
looks :+1: to me

**Pull request merged**